### ISSUE TITLE
Add a line wrapping option for code blocks

### ISSUE THREAD

**Issue by iangithubusername** [MacDownApp/macdown#1136]
I'd like there to be an option to have code blocks be line wrapped. Maybe a check box for it in Rendering -> "Syntax highlighted code block" next to "Show line numbers" etc.?

If I try hard enough I can dig through the css files in Contents/Resources/Prism/themes and change "white-space: pre;" to "white-space: pre-wrap;", but doing so breaks the code signature of the app. And that's not a very nice way to do it anyway.

**Comment by FranklinYu** [comment]
> doing so breaks the code signature of the app

I don’t think we currently have code signature (in macOS flavor) anyway… The main problem (I guess) would be your modification being overwritten during upgrade.

This is nice and PR welcome. I don’t have time for it recently.
